src/routes/+layout.svelte

```diff
@@ -36,13 +36,15 @@
   }
   .wrapper {
     display: flex;
+    width: 100%;
     flex: 1;
-    padding-top: 280px;
+    padding-top: 240px;
```

Comment:
magic number, where did u get it?

Reply:
It is the necessary number so that the logo is a little above, it takes less time because I added padding, I did it manually, you cannot add top or anything like that because the position is static.

Reply:
But these magic numbers only fit 1 screen and are not responsive

Reply:
ok i try to change this